helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -595,3 +613,8 @@ starrocksFeProxySpec:
   # ReadinessProbeFailureSeconds defines the total failure seconds of readiness Probe.
   # default value is 15 seconds
   readinessProbeFailureSeconds:
+  # Note: will create emptyDir volume for fe proxy, PVC is not supported.
+  emptyDirs: []
```

Comment:
what if the `emptyDirs` has a path that conflicts with the storageSpec path?

Reply:
that's what I am afraid of. statefulset refuses the change but the error will be silent to the user, the user is waiting for something happen but never happens, check if we can validate this kind of errors in operator?

Reply:
OK, a good suggestion, will check it. And we can also show errors(or k8s event) from statefulset and pod  on StarRocksCluster CR. So we can describe the StarRocksCluster CR to get the error.

Reply:
right! When the mount path conflicts from multiple configurations, what we can do for now is to refuse the crd apply, or show errors in cr status at least.

Reply:
Operator can not refuse the crd apply, we should use k8s webhook to achive this.